account_statement_import_sftp/components/account_statement_import_process.py

```diff
@@ -0,0 +1,29 @@
+# Copyright 2023 Compassion CH
+# @author: Simon Gonzalez
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+from odoo.addons.component.core import Component
+
+
+class EdiBankStatementImportProcess(Component):
+    _name = "edi.input.process.bank.statement.import"
+    _usage = "input.process"
+    _backend_type = "bk_sftp"
+    _inherit = "edi.component.input.mixin"
+
+    def process(self):
+        statement_import = self.env["account.statement.import"].create(
+            [
+                {
+                    "statement_file": self.exchange_record.exchange_file,
+                    "statement_filename": self.exchange_record.exchange_filename,
+                }
+            ]
+        )
+        action = statement_import.import_file_button()
+        if not action:
+            raise ValueError(f"The import hasn't been achieved. {statement_import}")
```

Comment:
Logging the statement_import object is useless in my opinion. Using the translation function is better for the user.
```suggestion
            raise ValueError(_("The import didn't succeed."))
```